bootstrap/httpserver/server_test.go

```diff
@@ -0,0 +1,207 @@
+package httpserver
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/orbs-network/go-mock"
+	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/orbs-network/orbs-spec/types/go/protocol/client"
+	"github.com/orbs-network/orbs-spec/types/go/services"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+	"github.com/orbs-network/orbs-network-go/test"
+)
+
+func Test_HttpServer_ReadInput_EmptyPost(t *testing.T) {
```

Comment:
underscores are a bad practice, except - maybe - to separate SUT from test case (before `EmptyPost` in your case).